docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+services:
+  nginx:
+    build:
+      context: .
+      dockerfile: nginx.dockerfile
+    image: uniengine/nginx
+    restart: unless-stopped
+    depends_on:
+      - php
+    ports:
+      - 80:80
```

Comment:
For local environment, it might be better to expose the containerized server on a different port than `80`, as some people will most likely have that port already taken by something else. I'm not sure if using env vars is possible here, but a simple random value above `1024` should be a bit better.

This could also be documented in README, that if anyone wish to use that compose in production, they should change this value to expose that on port `80`.

Reply:
I extracted these configs into environment variables, there are clear steps in the readme to ask the user to check and modify these configs